website/docs/index.html.markdown

```diff
@@ -141,3 +153,4 @@ The following arguments are supported:
 * `conn_params` - (Optional) Sets extra mysql connection parameters (ODBC parameters). Most useful for session variables such as `default_storage_engine`, `foreign_key_checks` or `sql_log_bin`.
 * `authentication_plugin` - (Optional) Sets the authentication plugin, it can be one of the following: `native` or `cleartext`. Defaults to `native`.
 * `iam_database_authentication` - (Optional) For Cloud SQL databases, it enabled the use of IAM authentication. Make sure to declare the `password` field with a temporary OAuth2 token of the user that will connect to the MySQL server.
+* `private_ip` - (Optional) Whether to use a connection to an instance with a private ip. Defaults to `false`.
```

Comment:
Please add that this applies to CloudSQL and it's ignored elsewhere.

Then I'll merge.

Reply:
Done!

I really appreciate that you consider merging my PR.